av1an-core/src/broker.rs

```diff
@@ -193,6 +206,20 @@ impl<'a> Broker<'a> {
           // avoids double-print of the error message as both a WARN and ERROR,
           // since `Broker::encoding_loop` will print the error message as well
           warn!("Encoder failed (on chunk {}):\n{}", chunk.index, e);
+
+          if self.project.encoder == Encoder::aom
+            && memmem::rfind_iter(e.stderr.as_bytes(), b"av1_tpl_stats_ready")
```

Comment:
Very small and pedantic nitpick but it appears that it may be slightly simpler to just do `memmem::rfind(..).is_some()` rather than `rfind_iter`. It also appears that `rfind` as opposed to `rfind_iter` uses the Rabin-Karp algorithm for small (<64 bytes) haystacks, otherwise uses the fallback impl.